gcloud-java-contrib/gcloud-java-nio-examples/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.gcloud</groupId>
```

Comment:
Removed here and in gcloud-java-nio. Also reformatted with a 2-space indent rule.